tests/lint_requirements.txt

```
@@ -5,3 +5,4 @@ pyflakes==2.2.0
 isort==5.2.1   # keep in sync with .pre-commit-config.yaml
 black==19.10b0
 flake8-bugbear
+flynt==0.50
```

```suggestion
flynt==0.50    # keep in sync with .pre-commit-config.yaml
```
and the mirror comment should be in `.pre-commit-config.yaml`

---

The issue is that flynt does not ship it's own `pre-commit-hooks.yml` file, so that I don't know how to pin an exact version of flynt. The pre-commit hook instead relies on a local installation of flynt and I don't know if we can force a specific version of flynt this way.

---

I'm on it

---

https://github.com/ikamensh/flynt/pull/73 🎉

---

That should be good now!